packages/vue/src/stories/releases/v0.11.x/v0.11.1/v0.11.1.stories.mdx

```diff
@@ -9,6 +9,7 @@ import { Meta } from "@storybook/addon-docs/blocks";
 
 ## 🚀 Features
 
+- SfButton: add aria-label to icon only / image only button
```

Comment:
Probably it will be better if you put here components names where these changes are applied :) so `SfGallery, SfProductCard, SfSearchBar: explanation of your changes`